README.rst

```diff
@@ -0,0 +1,591 @@
+down. A great place to start is the `trio docs`_ and this `blog post`_
+by njsmith_.
+
+``tractor`` takes much inspiration from pulsar_ and execnet_ but attempts to be much more
+minimal, focus on sophistication of the lower level distributed architecture,
+and of course does **not** use ``asyncio``, hence **no** event loops.
+
+.. _trio docs: https://trio.readthedocs.io/en/latest/
+.. _pulsar: http://quantmind.github.io/pulsar/design.html
+.. _execnet: https://codespeak.net/execnet/
+.. _blog post: https://vorpus.org/blog/notes-on-structured-concurrency-or-go-statement-considered-harmful/
+.. _njsmith: https://github.com/njsmith/
+
+
+A trynamic first scene
+----------------------
+As a first example let's spawn a couple *actors* and have them run their lines:
+
+.. code:: python
+
+    import tractor
+    from functools import partial
+
+    _this_module = __name__
+    the_line = 'Hi my name is {}'
+
+
+    async def hi():
+        return the_line.format(tractor.current_actor().name)
+
+
+    async def say_hello(other_actor):
+        await trio.sleep(0.4)  # wait for other actor to spawn
+        async with tractor.find_actor(other_actor) as portal:
+            return await portal.run(_this_module, 'hi')
+
+
+    async def main():
+        """Main tractor entry point, the "master" process (for now
+        acts as the "director").
+        """
+        async with tractor.open_nursery() as n:
+            print("Alright... Action!")
+
+            donny = await n.start_actor(
+                'donny',
+                main=partial(say_hello, 'gretchen'),
+                rpc_module_paths=[_this_module],
+                outlive_main=True
+            )
+            gretchen = await n.start_actor(
+                'gretchen',
+                main=partial(say_hello, 'donny'),
+                rpc_module_paths=[_this_module],
+            )
+            print(await gretchen.result())
+            print(await donny.result())
+            await donny.cancel_actor()
+            print("CUTTTT CUUTT CUT!!?! Donny!! You're supposed to say...")
+
+
+    tractor.run(main)
+
+
+Here, we've spawned two actors, *donny* and *gretchen* in separate
+processes. Each starts up and begins executing their *main task*
+defined by an async function, ``say_hello()``.  The function instructs
+each actor to find their partner and say hello by calling their
+partner's ``hi()`` function using a something called a *portal*. Each
+actor receives a response and relays that back to the parent actor (in
+this case our "director").
+
+To gain more insight as to how ``tractor`` accomplishes all this please
+read on!
+
+
+Actor spawning and causality
+----------------------------
+``tractor`` tries to take ``trio``'s concept of causal task lifetimes
+to multi-process land. Accordingly ``tractor``'s actor nursery behaves
+similar to the nursery_ in ``trio``. That is, an ``ActorNursery``
+created with ``tractor.open_nursery()`` waits on spawned sub-actors to
```

Comment:
Invert this sentence: "tractor.open_nursery creates ActorNursery". It moves to active voice rather than passive voice, making the chain of actions clearer.

You've also just introduced that you're talking about the trio nurseries here, so also don't blindly jump to ActorNursery - transition to it instead.